nodejs/src/index.js

```diff
@@ -49,10 +49,11 @@ const Cli = (inputOptions = CLIOPTIONS, inputArgs = process.argv) => {
 			`    independentGradient: ${args['independent-gradient']},\n` +
 			`    transitionGradient: ${args['transition-gradient']},\n` +
 			`    env: ${args.env},\n` +
-			`  }, ${args.debug}, ${args.debugLevel} );`,
+			`    rawMode: ${args['raw-mode']},\n` +
+			`  }, ${args.debug}, ${args['debug-level']} );`,
```

Comment:
Wait was this a bug? Did it mention the wrong casing here?

Reply:
Apparently. The args variable is a map created based off of CLIOPTIONS, so the keys are based on the cli flags, as opposed to the Options object used in other parts of the code.